### PR TITLE
feat: add client message, email-setting, email-template, discussion, notification, template-variable

### PR DESCRIPTION
**Pull request by tycloud97**
- add message-client
 - add email-setting-client
 - add email-template-client
 - add discussion-client
 - add notification-client
 - add template-variables-client

### CODE REVIEW

File: clients/email-setting-client/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# `@epilot/email-settings-client`
```
Comment (anttiviljami) [inline clients/email-setting-client/README.md:1]: Directory name should be `client/email-settings-client`

File: clients/email-setting-client/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+API Client for epilot email-settings API.
```
Comment (anttiviljami) [inline clients/email-setting-client/README.md:3]:
```suggestion
API Client for epilot Email Settings API.
```

File: clients/email-setting-client/package.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@epilot/email-settings-client",
+  "version": "0.0.3",
+  "description": "API Client for epilot email-settings API",
```
Comment (anttiviljami) [inline clients/email-setting-client/package.json:4]:
```suggestion
  "description": "API Client for epilot Email Settings API",
```

File: clients/email-setting-client/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+## BaseURL & Authorization
+
+To pass an authorization header and set up the API url, you can use axios
+defaults:
+
+```typescript
+const client = getClient();
+client.defaults.baseURL = config.API_URL;
+client.defaults.headers['authorization'] = `Bearer ${token}`;
+```
```
Comment (jpinho) [inline clients/email-setting-client/README.md:39]: authentication should be done via `authorizeWithToken`, see the entity client example below.

```
import { authorizeWithToken } from 'epilot-sdk/auth';
import entityClient from 'epilot-sdk/entity-client';

// authorize client using an access token
authorizeWithToken(entityClient, '<my_access_token>');
```

File: clients/notification-client/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import { getClient } from '@epilot/notification-client';
+```
+
+Use the client:
+```typescript
+// get typed client
+const client = await getClient();
+
+// call an operation
+const res = await client.testS3();
```
Comment (anttiviljami) [inline clients/notification-client/README.md:27]: Please update the example

File: clients/email-setting-client/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import { getClient } from '@epilot/email-settings-client';
+```
+
+Use the client:
+```typescript
+// get typed client
+const client = await getClient();
+
+// call an operation
+const res = await client.testS3();
```
Comment (anttiviljami) [inline clients/email-setting-client/README.md:27]: Please update the code example

File: clients/template-variable-client/README.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+# `@epilot/template-variables-client`
+
+API Client for epilot template-variables API.
```
Comment (jpinho) [inline clients/template-variable-client/README.md:3]:
```suggestion
API Client for epilot Template Variables API.
```

Comment (anttiviljami) [inline clients/template-variable-client/README.md:3]:
```suggestion
API Client for epilot Template Variables API.
```

File: clients/notification-client/README.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+# `@epilot/notification-client`
+
+API Client for epilot notification API.
```
Comment (jpinho) [inline clients/notification-client/README.md:3]:
```suggestion
API Client for epilot Notification API.
```

File: clients/template-variable-client/README.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
+# `@epilot/template-variables-client`
```
Comment (anttiviljami) [inline clients/template-variable-client/README.md:1]: Directory should match the name of the client package

`clients/template-variables-client`

File: clients/template-variable-client/README.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+## BaseURL & Authorization
+
+To pass an authorization header and set up the API url, you can use axios
+defaults:
+
+```typescript
+const client = getClient();
+client.defaults.baseURL = config.ENTITY_API_URL;
+client.defaults.headers['authorization'] = `Bearer ${token}`;
+```
```
Comment (jpinho) [inline clients/template-variable-client/README.md:39]: Please update with `authorizeWithToken`

File: clients/template-variable-client/README.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+import { getClient } from '@epilot/template-variables-client';
+```
+
+Use the client:
+```typescript
+// get typed client
+const client = await getClient();
+
+// call an operation
+const res = await client.testS3();
```
Comment (anttiviljami) [inline clients/template-variable-client/README.md:27]: Please update the example

File: packages/epilot-sdk/src/template-variable-client.ts [hunk 12]
```diff
@@ -0,0 +1,6 @@
+import { getClient } from '@epilot/template-variables-client';
```
Comment (anttiviljami) [inline packages/epilot-sdk/src/template-variable-client.ts:1]: Module name should match the package name.

`packages/epilot-sdk/src/template-variables-client.ts`

File: packages/epilot-sdk/src/email-setting-client.ts [hunk 13]
```diff
@@ -0,0 +1,6 @@
+import { getClient } from '@epilot/email-settings-client';
```
Comment (anttiviljami) [inline packages/epilot-sdk/src/email-setting-client.ts:1]: Module name should match the  package name

`packages/epilot-sdk/src/email-setting-client.ts`

File: clients/email-template-client/README.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
+# `@epilot/email-template-client`
+
+API Client for epilot email template API.
```
Comment (jpinho) [inline clients/email-template-client/README.md:3]:
```suggestion
API Client for epilot email Template API.
```

**Pull request merged**